src/Lecture3.hs

```diff
@@ -1,4 +1,5 @@
 {-# LANGUAGE InstanceSigs #-}
+{-# OPTIONS_GHC -Wno-incomplete-patterns #-}
```

Comment:
Ideally, this warning shouldn't be disabled. Such things need to be fixed and not suppressed 🙂

Reply:
Ok. I don't know how this appears )))